CHANGES.md

```diff
@@ -6,6 +6,10 @@
     Placement of documentation comments on `val` and `external` items is now
     controled by `doc-comments` unless this option is set.
 
+#### Bug fixes
+
+  + Preserve unwrapped comments by not adding artificial breaks when `wrap-comments=false` and `ocp-indent-compat=true` are set to not interfere with ocp-indent indentation. (#1352) (Guillaume Petiot)
```

Comment:
```suggestion
  + Preserve unwrapped comments by not adding artificial breaks when `wrap-comments=false` and `ocp-indent-compat=true` are set to avoid interfering with ocp-indent indentation. (#1352) (Guillaume Petiot)
```